pom.xml

```diff
@@ -107,6 +107,22 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>3.4</version>
 		</dependency>
+		<dependency>
+		<!-- Fix java.lang.NoSuchMethodError: org.apache.commons.io.IOUtils.toString(Ljava/io/InputStream;Ljava/nio/charset/Charset;)Ljava/lang/String; -->
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.6</version>
+		</dependency>
+		<dependency>
+			<groupId>de.retest</groupId>
+			<artifactId>recheck</artifactId>
```

Comment:
Is recheck used here?

Reply:
Das liefert den Pixel-Diff Code.

Reply:
`ExactImageDifferenceCalculator`

Reply:
Hab keine bessere Java-Implementierung gefunden, auf die Schnelle. Wenn Du noch was besseres hast, immer her damit ;-)